### PR TITLE
[Client encryption]: Address code styling errors and warnings

### PR DESCRIPTION
**Pull request by JanHyka**
# Pull Request Template

## Description

Cleanup of code styling issues in Cosmos.Encryption.Custom.* projects as set up by editorconfig.

## Type of change

Please delete options that are not relevant.

- [] Bug fix (non-breaking change which fixes an issue)

## Closing issues

Contributes to #4678

### CODE REVIEW

File: Microsoft.Azure.Cosmos.Encryption.Custom/src/DataEncryptionKeyFeedIterator{T}.cs [hunk 1]
```diff
@@ -20,11 +20,6 @@ public DataEncryptionKeyFeedIterator(
             DataEncryptionKeyFeedIterator feedIterator,
             CosmosResponseFactory responseFactory)
         {
-            if (!(feedIterator is DataEncryptionKeyFeedIterator))
```
Comment (JanHyka) [inline Microsoft.Azure.Cosmos.Encryption.Custom/src/DataEncryptionKeyFeedIterator{T}.cs:23]: Note DateEncryptionKeyFeedIterator is `sealed`, this validation is implicitly broken. Was the intention here null check perhaps?

**Review by kirankumarkolli — APPROVED** [review]

File: Microsoft.Azure.Cosmos.Encryption.Custom/src/DataEncryptionKeyFeedIterator{T}.cs (hunk 1 above)
Comment (kirankumarkolli) [reply Microsoft.Azure.Cosmos.Encryption.Custom/src/DataEncryptionKeyFeedIterator{T}.cs:23]: Can we please add null check here instead.

Comment (JanHyka) [reply Microsoft.Azure.Cosmos.Encryption.Custom/src/DataEncryptionKeyFeedIterator{T}.cs:23]: Added missing null checks + tests

**Pull request merged**